ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -35,7 +35,7 @@ Rectangle {
     property var emojiPopup: null
     // Use this to only enable the Connections only when this Input opens the Emoji popup
     property bool emojiPopupOpened: false
-    property bool closeGifPopupAfterSelection: false
+    property bool closeGifPopupAfterSelection: true
```

Comment:
Wonder why this was false in the first place..

Reply:
Oh and those callbacks :open_mouth:

Reply:
The boolean was introduced here: a70d800e6dd498161bdff3b46c0570c87e792b53